infoschema/tables.go

```diff
@@ -1110,14 +1110,31 @@ func GetTiDBServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
 			ServerType:     "tidb",
 			Address:        fmt.Sprintf("%s:%d", node.IP, node.Port),
 			StatusAddr:     fmt.Sprintf("%s:%d", node.IP, node.StatusPort),
-			Version:        node.Version,
+			Version:        FormatVersion(node.Version),
 			GitHash:        node.GitHash,
 			StartTimestamp: node.StartTimestamp,
 		})
 	}
 	return servers, nil
 }
 
+// FormatVersion make TiDBVersion consistent to TiKV and PD.
+func FormatVersion(TiDBVersion string) string {
+	var version string
+	nodeVersion := TiDBVersion[strings.LastIndex(TiDBVersion, "TiDB-")+len("TiDB-"):]
```

Comment:
Check `len(TiDBVersion) > n` before use

Reply:
+1.

Since the `TiDBVersion` maybe equal the config `ServerVersion`, Add more case for this, such as `xxx`

Reply:
Test it need change the `serverVersion` config, so test it locally and show the result in  `introduction`.
1. When not set the `serverVersion`, it would use the default version, and format it in `cluster_info`.
2. When the user set the `serverVersion`, it would use the `serverVersion` and not do the format.